packages/cli/src/evaluation.ee/test-runner/test-runner.service.ee.ts

```diff
@@ -280,8 +283,7 @@ export class TestRunnerService {
 				// In case of an unexpected error, increment the failed count and continue with the next test case
 				await this.testRunRepository.incrementFailed(testRun.id);
 
-				// Report error to Sentry
-				captureException(e);
+				this.errorReporter.error(e);
```

Comment:
Is there any chance the execution might fail due to user misconfiguring things? We already get a lot of spam into sentry, so we should avoid adding anymore that could potentially be user caused

Reply:
As far as I understand – errors originating in the workflow under test or evaluation workflow will not throw an exception, and will have the original exception stored in `data.resultData.error` instead, so it will be handled without sending an event to Sentry in the code above.